middleware/index.js

```diff
@@ -1,39 +1,38 @@
-const Campground = require('../models/campground')
+const Article = require('../models/article')
 const Comment = require('../models/comment')
 //all the middleware goes here
 let middlewareObj = {}
 
-middlewareObj.checkCampgroundOwnership = (req, res, next) => {
-    if(req.isAuthenticated()){
-        Campground.findById(req.params.id, (err, foundCampground) => {
+middlewareObj.checkArticleOwnership = (req, res, next) => {
+    if(req.isAuthenticated()) {
```

Comment:
I'd also personally do early return, instead of closing whole body in a condition, to get rid of unecessary nesting.
That is, 
```
if (!req.isAuthenticated()) {
    ...code for error handling...
   return
}
... rest of the code if authenticated
```

Reply:
Fixed.